bears/haml/HAMLLintBear.py

```diff
@@ -0,0 +1,396 @@
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+from coalib.settings.Setting import typed_list
+from dependency_management.requirements.GemRequirement import GemRequirement
+import yaml
```

Comment:
Please put this import at the top and separate it from the others with an empty line :)
```
import yaml

from coalib....
```